message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -6,6 +6,7 @@
   <include>icarous.xml</include>
   <include>loweheiser.xml</include>
   <include>cubepilot.xml</include>
+  <include>csAirLink.xml</include>
```

Comment:
This xml also needs to be in all.xml - with the range allocated specified in the comments etc etc.  That patch also needs to go upstream - you don't want someone else tromping into your allocated space!